packages/ui/src/components/Badge/Badge.helper.ts

```diff
@@ -1,6 +1,6 @@
 import { CSSObject } from '@emotion/react'
-import { Level, mq } from '../../lib/media-query'
-import { theme } from '../../lib/theme/theme'
+import { theme } from '../../theme'
+import { Level, mq } from '../../theme/media-query'
```

Comment:
Since everything is _exportable_ from `theme` should we condense those into a single import statement?

```suggestion
import { theme, mq, type Level } from '../../theme'
```